eng/common/PublishBuildAssets.cmd

```diff
@@ -1,3 +1,3 @@
 @echo off
```

Comment:
These changes to eng/common will get overwritten. We're they diagnostic for your workflow, or should we wrap them?

Reply:
Is there a chance to revert them?

Reply:
These were just copied from the latest arcade. Let me fix the merge conflicts, and then if there's not a significant difference between the latest and what's already checked in to coreclr, I'll remove the changes from this pr.

Reply:
I'm going to keep the manual arcade update because it contains a fix that we need (copied from arcade - I fully expect dependency flow to overwrite this with an even more recent version).